assets/src/scss/new-identity/style.scss

```diff
@@ -121,18 +121,41 @@ table.spreadsheet-table.is-color-gp-green {
   }
 }
 
-a:where(.standalone-link) {
+a:where(.standalone-link, .split-two-column-item-link) {
```

Comment:
Why don't we directly write `.standalone-link, .split-two-column-item-link` here? Is there an advantage to adding `a:where` in front? 🤔

Reply:
I meant why not write directly `.standalone-link, .split-two-column-item-link { ... }` instead of `a:where(.standalone-link, .split-two-column-item-link) { ... }`, basically I'm not sure I understand why we add `a:where`